File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    var strArr = s.split(" ")
    
    for (var strItem of strArr){
        for(var idx in strItem){
            if(idx%2 == 0) answer += strItem[idx].toUpperCase()
            else answer += strItem[idx].toLowerCase()
        }
        answer += ' '
    }
    return answer.slice(0, s.length);
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    strArr = s.split(" ")
    
    for strItem in strArr:
        for idx in range(0, len(strItem)):
            if(idx%2 == 0):
                answer += strItem[idx].upper()
            else:
                answer += strItem[idx].lower()
        answer += ' '
        
    return answer[0:len(s)]